examples/auto-test/start_all_test.py

```diff
@@ -101,7 +101,7 @@ def scan(self, rootpath):
             if os.path.isdir(rootpath + '/' + item):
                 files = os.listdir(rootpath + '/' + item)
                 if "prepare.sh" in files:  # run the prepare shell script to download the data and code, then upload them to hdfs
-                    os.system("/bin/bash " + rootpath + '/' + item + '/' + "prepare.sh " + self.config.hdfs_url)
+                    os.system("/bin/bash " + rootpath + '/' + item + '/' + "prepare.sh " + self.config.hdfs_url + " " + self.config.PAI_username)
```

Comment:
I think it's unnecessary, because `os.system("/bin/bash " + os.path.join(rootpath, item) + " " + self.config.hdfs_url + " " + self.config.PAI_username) ` is not more simple.